### DISCUSSION TITLE
The scalar columns  and vector columns have different update frequencies

### DISCUSSION THREAD

**Discussion by YolandaLyj** [milvus-io/milvus#35903] [Q&A and General discussion] [answered]
I have some scalar columns and some vector columns. 

The scalar column is updated frequently, and I don't want to add vector data every time when I update the property. 

Any suggestions for this situation？

**Comment by yhmo** [comment, marked as answer]
We have a plan to support partly updating entities in Milvus v3.0. before the v3.0 is released, the upsert() interface is the only interface to update entities, which requires inputting the entire row data.

**Reply by YolandaLyj** [reply]
Can you explain the implementation details of partly updating entities？

**Reply by yhmo** [reply]
The technical design has not begun, yet.
